cirq-superstaq/cirq_superstaq/job.py

```diff
@@ -319,6 +320,37 @@ def input_circuits(
         """
         return self._get_circuits("input_circuit", index=index)
 
+    def pulse_gate_circuits(self) -> Any:
+        """Gets the pulse gate circuit returned by this job.
+
+        Returns:
+            The `qiskit.QuantumCircuit` pulse gate circuit.
+
+        Raises:
+            ValueError: If job was not run on an IBM pulse device.
+        """
+        job_ids = self._job_id.split(",")
+
+        if not all(
+            job_id in self._job and self._job[job_id].get("pulse_gate_circuits")
+            for job_id in job_ids
+        ):
+            self._refresh_job()
+
+        if all(self._job[job_id].get("pulse_gate_circuits") for job_id in job_ids):
+            serialized_circuits = [self._job[job_id]["pulse_gate_circuits"] for job_id in job_ids]
+            deserialized_circuits = []
+            for serialized_circuit in serialized_circuits:
+                deserialized_circuit = css.serialization.deserialize_qiskit_circuits(
+                    serialized_circuit, False
+                )
+                if deserialized_circuit is not None:
+                    deserialized_circuits.append(deserialized_circuit[0])
+            return deserialized_circuits
+
+        error = f"Job: {self._job_id} was not submitted to a pulse-enabled device"
```

Comment:
wdyt about changing this to something like
```suggestion
        error = f"Target '{self.target()}' does not use pulse gate circuits."
```
? some targets might be pulse-enabled (e.g. via braket) but not use qiskit pulse gate circuits